samples/BenchmarkApp/Program.cs

```diff
@@ -19,5 +23,25 @@ public static void Main(string[] args)
                 {
                     webBuilder.UseStartup<Startup>();
                 });
+
+        private static void WriteStatistics()
+        {
+            Console.WriteLine("#StartJobStatistics");
+
+            Console.WriteLine(JsonSerializer.Serialize(new {
+                Metadata = new object[]
+                {
+                    new { Source= "Benchmarks", Name= "AspNetCoreVersion", ShortDescription = "ASP.NET Core Version", LongDescription = "ASP.NET Core Version" },
+                    new { Source= "Benchmarks", Name= "NetCoreAppVersion", ShortDescription = ".NET Runtime Version", LongDescription = ".NET Runtime Version" },
+                },
+                Measurements = new object[]
+                {
+                    new { Timestamp = DateTime.UtcNow, Name = "AspNetCoreVersion", Value = typeof(WebHostBuilder).GetTypeInfo().Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion },
```

Comment:
```suggestion
                    new { Timestamp = DateTime.UtcNow, Name = "AspNetCoreVersion", Value = typeof(IWebHostBuilder).GetTypeInfo().Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>()?.InformationalVersion },
```
We plan to obsolete WebHostBuilder.